src/app/static/src/tests/components/files/manageFile.test.ts

```diff
@@ -14,17 +13,14 @@ import {expectTranslated} from "../../testHelpers";
 
 describe("Manage file component", () => {
 
-    const mockGetters = {
-        editsRequireConfirmation: () => false
-    };
 
-    const createStore = () => {
+    const createStore = (customStore: any = emptyState(), requireConfirmation = false) => {
```

Comment:
cant this type not be `DataExplorationState`? and similarly in other test suites above